libs/clipboard/src/test.ts

```diff
@@ -1,21 +1,31 @@
 // This file is required by karma.conf.js and loads recursively all the .spec and framework files
-import 'zone.js/dist/zone';
-import 'zone.js/dist/zone-testing';
+import 'zone.js';
+import 'zone.js/testing';
```

Comment:
I copied this from what Angular CLI 14 generates today (our current edition of `test.ts` is a bit outdated).